### PR TITLE
CSS 2021 figures

### PR DESCRIPTION
**Pull request by rviscomi**
Progress on #2140 

Adding figures to the empty markdown file and saving the corresponding screenshots

cc @meyerweb

### CODE REVIEW

**Comment by rviscomi** [comment]
Assigning to @meyerweb to add figure descriptions

File: src/content/en/2021/css.md [hunk 1]
```diff
@@ -18,3 +18,1098 @@ featured_stat_label_3: TODO
...
+) }}
+
+
+
+{{ figure_markup(
+  caption="The heaviest total weight of images loaded via CSS, in KB.",
+  content="314,386",
+  classes="big-number",
+  sheets_gid="TODO",
+  sql_file="TODO"
```
Comment (rviscomi) [inline src/content/en/2021/css.md:611]:
```suggestion
  sheets_gid="361647805",
  sql_file="image_weights.sql"
```

**Review by rviscomi — COMMENTED** [review]
@meyerweb added the big number metadata in code review to avoid causing a merge conflict for you

File: src/static/css/page.css [hunk 2]
```diff
@@ -612,6 +612,14 @@ td.numeric {
   text-align: right;
 }
 
+td .numeric-good {
+  color: #38761d;
+}
+
+td .numeric-bad {
+  color: #e00;
```
Comment (tunetheweb) [inline src/static/css/page.css:620]: Can we make this a darker red to pass Lighthouse Accessibility tests?

```suggestion
  color: ##B60000;
```

**Comment by rviscomi** [comment]
@meyerweb is it ok with you if we merge this? You can open a new PR with the content integrations and any edits to the figure descriptions. Let me know if there are any changes I can help with on the figures themselves.

**Comment by meyerweb** [comment]
@rviscomi Thumbs up from me!

**Pull request merged**